src/User.java

```diff
@@ -211,4 +213,14 @@ public CurrentBook getSpecificCurrentBook(int bookIDNum){
         return new CurrentBook(-1, "", "", "");
     }
 
+    //NEW SPRINT3
```

Comment:
removes BookGroup from the bookGroups attribute (it is removed from other places by function in other file)